actionview/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Updated `@rails/ujs` files to follow https://github.com/rails/rails/commit/8e3449908c59858384ae230d1416c7dcabc8c2dc.
```

Comment:
Changelog entries are for users, not for framework developer/contributors. So would you rewrite the link to the commit for more meaningful message.